tests/python_tests/conftest.py

```diff
@@ -198,13 +208,46 @@ def pytest_runtest_makereport(item, call):
 
                 test_file_and_func = report.nodeid.split("[")[0]
 
+                stack_trace = []
+                for entry in call.excinfo.traceback:
+                    path_str = str(entry.path)
+                    # Skip pytest and pluggy internal frames
+                    if "_pytest" in path_str or "pluggy" in path_str:
+                        continue
+
+                    try:
+                        file_path = entry.path.relative_to(Path)
```

Comment:
```suggestion
                        file_path = entry.path.relative_to(Path.cwd())
```
Would be correct. And we would still be calling it in every loop iteration. Can it be hoisted out?

Reply:
It makes very usefull prints in the error messages that are not so verbose. Will fix